src/encoded/static/components/item-pages/components/WorkflowDetailPane/FileDetailBody.js

```diff
@@ -0,0 +1,303 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import { Fade } from 'react-bootstrap';
+import { console, object, layout, ajax, fileUtil, expFxn } from './../../../util';
+import { getTitleStringFromContext } from './../../item';
+import { FlexibleDescriptionBox } from './../FlexibleDescriptionBox';
+import { SimpleFilesTable } from './../SimpleFilesTable';
+import { ItemDetailList } from './../ItemDetailList';
+import { ExperimentSetTablesLoaded } from './../ExperimentSetTables';
+import { ViewMetricButton, MetricsView } from './FileDetailBodyMetricsView';
+
+
```

Comment:
DetailPanes are not super important at moment and will likely change in future.

Currently works somewhat like decision tree of components.
<WorkflowDetailPane/> renders one of [FileDetailBody, StepDetailBody, WFRDetailBody, ParameterDetailBody]

Each of those reads data off of selectedNode.meta for everything it needs. Will perform AJAX of @id to get more info, etc. If is QC metric, FileDetailBody will render some additional rows for QC stuff, etc.